readme.md

```diff
@@ -78,6 +78,27 @@ All browsers that support [`Array.isArray`](https://developer.mozilla.org/en-US/
 
 >**Note:** For IE8 support and older, please install `clsx@1.0.x` and beware of [#17](https://github.com/lukeed/clsx/issues/17).
 
+## Tailwind Support
+
+Here some additional (optional) steps to enable classes autocompletition using `clsx` with Tailwind CSS.
+<details>
+
+<summary>
+  Visual Studio Code
+</summary>
+
+1. [Install the "Tailwind CSS IntelliSense" Visual Studio Code extension](https://marketplace.visualstudio.com/items?itemName=bradlc.vscode-tailwindcss)
+
+2. Add the following to your [`settings.json`](https://code.visualstudio.com/docs/getstarted/settings):
+
+  ```json
+   {
+    "tailwindCSS.experimental.classRegex": [
+      ["clsx\\(([^)]*)\\)", "(?:'|\"|`)([^']*)(?:'|\"|`)"]
```

Comment:
👋 @lukeed! Thanks for adding this regex! Can you please elaborate on your approach to building it? I was using https://github.com/shadcn/ui/blob/5e172fc34fdf015aa0d141f52cc8c082b8ae6613/.vscode/settings.json#L7-L8 before seeing your solution. There is a slight difference, so I'm curious if there are any edge cases not covered by any of them:

```diff
shadcn/ui
- ["cn\\(([^)]*)\\)", "[\"'`]([^\"'`]*).*?[\"'`]"]
+ ["cn\\(([^)]*)\\)", "(?:'|\"|`)([^']*)(?:'|\"|`)"]
your regex
```

Reply:
@edgarlr contributed this section so perhaps they can offer an explanation.

By eyeballing it, the difference is that `([^']*)` will match anything up to `'`, which means that if `"` or or backticks appear in the class value, then the regexp in _our_ docs will include it as part of the value, whereas the other regexp stops there.

clsx doc's regexp
<img width="283" alt="Screen Shot 2023-07-18 at 10 43 53 AM" src="https://github.com/lukeed/clsx/assets/5855893/0a0f4e46-08bc-4d09-9552-dbd594c5b699">

shadcn/ui regexp
<img width="264" alt="Screen Shot 2023-07-18 at 10 44 16 AM" src="https://github.com/lukeed/clsx/assets/5855893/415538d8-7435-42ed-b41d-0f1740d2238b">

Looks like they're both not ideal, tbh
I don't use this extension, but perhaps something like this is better? I don't know if the extn is looking for the first matching group only, though

<img width="218" alt="Screen Shot 2023-07-18 at 10 53 57 AM" src="https://github.com/lukeed/clsx/assets/5855893/94adef8d-a66b-41da-98ef-e64b3ed0a8ef">